src/coreclr/vm/eventing/eventpipe/ds-rt-coreclr.h

```diff
@@ -48,6 +48,9 @@
 #undef DS_LOG_DEBUG_0
 #define DS_LOG_DEBUG_0(msg) STRESS_LOG0(LF_DIAGNOSTICS_PORT, LL_INFO1000, msg)
 
+#undef DS_LOG_DEBUG_1
```

Comment:
Couldn't find that this was used in PR (just defined), maybe it should be added for future or just a left over?

Reply:
If you would like to keep it (for future) it needs to be added to ds-rt-mono.h as well.

Reply:
ah, leftover from the histogram code I used during development.  I can just remove it.